src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int ONE = 1;
+    private static final int ZERO = 0;
```

Comment:
The constants `ONE` and `ZERO` are not meaningful and don't improve code readability. It's better to use the actual numbers in the code since `0` and `1` are not magic numbers in this context and are commonly used for array indexing.